### PR TITLE
[Docs] manual instrumentation

### PR DESCRIPTION
**Pull request by lachmatt**
Manual instrumentation docs.

### CODE REVIEW

File: docs/manual-instrumentation.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+For the list of steps required, see [link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
+
+## Instrument using OpenTracing API
+
+1. Add the `OpenTelemetry.Shims.OpenTracing` dependency to your project:
+
+    ```xml
+    <PackageReference Include="OpenTelemetry.Shims.OpenTracing" Version="1.0.0-rc9.10" />
+    ```
```
Comment (pellared) [inline docs/manual-instrumentation.md:18]: Is it not enough to reference `OpenTracing`?

File: docs/manual-instrumentation.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+The auto-instrumentation provides a base you can build on by adding your own
+custom instrumentation. By using both instrumentation approaches,
+you'll be able to present a more detailed representation of the logic
+and functionality of your application, clients, and framework.
+
+## Instrument using System.Diagnostics API
+
+For the list of steps required, see [link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
+
+## Instrument using OpenTracing API
```
Comment (pellared) [inline docs/manual-instrumentation.md:12]: I think we should add a hyperlink in `migrating.md` to this header.

File: docs/manual-instrumentation.md (hunk 1 above)
Comment (pellared) [reply docs/manual-instrumentation.md:18]: So we need also to reference `System.Diagnostics.DiagnosticSource` `7.0.0`.

**Comment by theletterf** [comment]
Will review this one next week if that's OK. :)

File: docs/manual-instrumentation.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+# Manually instrument a .NET application
+
+The auto-instrumentation provides a base you can build on by adding your own
```
Comment (theletterf) [inline docs/manual-instrumentation.md:3]:
```suggestion
The automatic instrumentation provides a base you can build upon by adding your own
```

File: docs/manual-instrumentation.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+custom instrumentation. By using both instrumentation approaches,
+you'll be able to present a more detailed representation of the logic
```
Comment (theletterf) [inline docs/manual-instrumentation.md:5]:
```suggestion
you can produce a more detailed representation of the logic
```

File: docs/manual-instrumentation.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+and functionality of your application, clients, and framework.
+
+## Instrument using System.Diagnostics API
+
+For the list of steps required, see [link](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
```
Comment (theletterf) [inline docs/manual-instrumentation.md:10]:
```suggestion
For the list of steps required, see the [upstream documentation](https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/blob/main/docs/manual-instrumentation.md).
```

Comment (theletterf) [reply docs/manual-instrumentation.md:10]: Any step that would be custom for our distribution?

File: docs/manual-instrumentation.md (hunk 2 above)
Comment (theletterf) [reply docs/manual-instrumentation.md:12]: Couldn’t we move the section to the migration section? It feels a bit strange to me that someone would use OpenTracing when a new Otel rostro is available. At the very minimum we should explain why would someone use OT here.

File: docs/manual-instrumentation.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+## Instrument using OpenTracing API
+
+1. Add the `OpenTelemetry.Shims.OpenTracing` dependency to your project:
+
+    ```xml
+    <PackageReference Include="OpenTelemetry.Shims.OpenTracing" Version="1.0.0-rc9.10" />
+    ```
+
+1. Set `OTEL_DOTNET_AUTO_OPENTRACING_ENABLED` environment variable to `true`
```
Comment (theletterf) [inline docs/manual-instrumentation.md:20]:
```suggestion
1. Set the `OTEL_DOTNET_AUTO_OPENTRACING_ENABLED` environment variable to `true`.
```

**Review by theletterf — CHANGES_REQUESTED** [review]

File: docs/manual-instrumentation.md (hunk 1 above)
Comment (lachmatt) [reply docs/manual-instrumentation.md]: changed in https://github.com/signalfx/splunk-otel-dotnet/pull/126/commits/8271315acf4cab6bd19f7fa0328e5e338bc7c28c
In version 0.5.0 of automatic instrumentation, no additional span is created (tags are added to existing span).
This seems to be fixed in latest versions of automatic instrumentation.

File: docs/manual-instrumentation.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+# Manually instrument a .NET application
```
[the remark this answers is not in the document]
Comment (lachmatt) [reply docs/manual-instrumentation.md:1]: clarified in https://github.com/signalfx/splunk-otel-dotnet/pull/126/commits/8271315acf4cab6bd19f7fa0328e5e338bc7c28c
`OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES` needs to be set if manual spans are created using `System.Diagnostics` API's (linked documentation from upstream mentions how to set `OTEL_DOTNET_AUTO_TRACES_ADDITIONAL_SOURCES` )
I added a separate section for custom metrics, and mentioned `OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES` there.

File: docs/manual-instrumentation.md (hunk 5 above)
Comment (lachmatt) [reply docs/manual-instrumentation.md:12]: No custom steps needed for our distribution.

**Review by theletterf — APPROVED** [review]

**Review by theletterf — APPROVED** [review]

**Review by Kielek — APPROVED** [review]

**Pull request merged**